scripts/phylum-init.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+
+set -eu
+
+SUPPORTED_TARGETS="aarch64-apple-darwin x86_64-apple-darwin x86_64-unknown-linux-musl"
+BASE_URL="https://github.com/phylum-dev/cli/releases/latest/download"
+MINISIG_PUBKEY="RWT6G44ykbS8GABiLXrJrYsap7FCY77m/Jyi0fgsr/Fsy3oLwU4l0IDf"
+
+# Get the platform name.
+get_platform() {
+    platform=$(uname -s | tr '[:upper:]' '[:lower:]')
+
+    case "${platform}" in
+        linux) platform="unknown-linux-musl" ;;
+        darwin) platform="apple-darwin" ;;
+    esac
+
+    echo "${platform}"
+}
+
+# Get the architecture
+get_arch() {
+    arch=$(uname -m | tr '[:upper:]' '[:lower:]')
+
+    case "${arch}" in
+        arm64) arch="aarch64" ;;
+        amd64) arch="x86_64" ;;
+    esac
+
+    echo "${arch}"
+}
+
+# Get the target triple
+get_target_triple() {
+    echo "$(get_arch)-$(get_platform)"
+}
+
+# Check if a target is supported
+is_supported() {
+    target="$1"
+
+    for tgt in ${SUPPORTED_TARGETS}; do
+        if [ "${target}" = "${tgt}" ]; then
+            return 0
+        fi
+    done
+
+    return 1
+}
+
+# Download the file to the given location
+download() {
+    curl --fail --silent --show-error --location --output "$1" "$2"
+}
+
+# Check for a required command
+require_command() {
+    cmd="$1"
+    help_msg="${2:-}"
+
+    if ! type "${cmd}" > /dev/null 2>&1; then
+        echo "ERROR: This script requires \`${cmd}\`. Please install it and re-run this script to continue." >&2
+        if [ -n "${help_msg}" ]; then
+            printf "\n" >&2
+            echo "${help_msg}" >&2
+        fi
+        exit 1
+    fi
+}
+
+# All of the required commands (outside of POSIX)
+require_command mktemp
+require_command curl
+require_command unzip
+
+# Parse command line arguments
```

Comment:
You have command line arguments here, but no way they're ever communicated to the user. Even if you don't want to add a dedicated `--help` command, I think we should at least log all the options whenever an unrecognized parameter is detected.

Reply:
It's simple enough to add `--help` if we're going to print usage for unrecognized parameters. Done.